cypress/integration/multi_team/system_message_spec.js

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+/* eslint max-nested-callbacks: ["error", 4] */
+
+function verifySystemMessage(post) {
+    cy.get(post).
+        invoke('attr', 'class').
+        should('contain', 'post--system').
+        should('not.contain', 'same--root').
+        should('not.contain', 'other--root').
+        should('not.contain', 'current--user').
+        should('not.contain', 'post--comment').
+        should('not.contain', 'post--root');
+}
+
+function verifyStatusIconNotVisible(post) {
```

Comment:
You can probably roll these 2 functions into one as just `verifySystemMessage` since they are not doing a ton of stuff, nor are being called more than once respectively.

Reply:
Done. Should I mark these as resolved or wait for you to do it once you've checked over the changes?